renterd/build/darwin/Info.plist

```diff
@@ -8,7 +8,7 @@
         <key>CFBundleExecutable</key>
         <string>{{.Name}}</string>
         <key>CFBundleIdentifier</key>
-        <string>com.siafoundation.renterd.{{.Name}}</string>
+        <string>com.wails.{{.Name}}</string>
```

Comment:
Can this go back to `siafoundation.renterd`?

Reply:
@n8maninger yeah, also feel free to reset `master` back to prev commit, this was WIP didn't mean to merge it yet lol.